lib/matplotlib/collections.py

```diff
@@ -394,7 +394,7 @@ def set_hatch(self, hatch):
         *hatch* can be one of::
 
           /   - diagonal hatching
-          \   - back diagonal
+          \\   - back diagonal
```

Comment:
This one worries me as it is something I would expect to be read either on the web or in the terminal.  Probably better to make this a `r'''` string?